### DISCUSSION TITLE
Card: Shock doctrine

### DISCUSSION THREAD

**Discussion by RebelusQuo** [RebelusQuo/the-wall-is-down#16] [Ideas]
So here's the reason for asking for clarification on the NWO track mechanic. When playing as West, you can just play this on the track directly in order to only remove Supremacy from one Slot. When benefiting by play by East you can also just pick the first option and vie for a Slot, or none by the principle of "do as much as possible" if you already have Supremacy over all (that you're allowed to) instead of removing two. In either case "up to" is not required.

Speaking of, this should have the reminder about following Aheads and Second Epoch limitations, right?

So I guess that leaves us with the following suggestion:

> Vie for Supremacy over one Economy Slot (following normal Aheads and Second Epoch limitations), *or* remove Supremacy over two Slots.

**Comment by rodrigoSantamaria** [comment]
In this case, it should not be 'vie for Supremacy' but 'gain supremacy'. This card not only removes but also adds your supremacy. If not, it's not very different to using the NWO standard op.
Other than that, the suggestion is ok to me.

**Comment by RebelusQuo** [comment]
Ah, I missed that. So:
> Gain direct Supremacy over one Economy Slot (following normal Aheads and Second Epoch limitations), **or** remove Supremacy over two Slots.

**Reply by rodrigoSantamaria** [reply]
Now is perfect

**Reply by RebelusQuo** [reply]
Acknowledged.
